### PR TITLE
[zh]Sync pod-overhead.md

### PR DESCRIPTION
**Pull request by mengjiao-liu**
related issue: #33476

concepts-1
```
12	14	content/en/docs/concepts/security/pod-security-admission.md
1	1	content/en/docs/concepts/security/controlling-access.md
3	9	content/en/docs/concepts/scheduling-eviction/assign-pod-node.md
48	46	content/en/docs/concepts/scheduling-eviction/pod-overhead.md
```
The first three files have been updated or submitted for PR.

- [ ] https://github.com/kubernetes/website/pull/33458
- [ ] https://github.com/kubernetes/website/pull/33497

So this PR only updates the file `content/en/docs/concepts/scheduling-eviction/pod-overhead.md`.

### CODE REVIEW

**Comment by tengqm** [comment]
Thanks.
/approve

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 1]
```diff
@@ -141,8 +137,7 @@ RuntimeClass 中定义的 `overhead`。如果 PodSpec 中已定义该字段，
 <!--
 After the RuntimeClass admission controller, you can check the updated PodSpec:
 -->
-在 RuntimeClass 准入控制器之后，可以检验一下已更新的 PodSpec:
-
+在 RuntimeClass 准入控制器进行修改后，你可以检验一下更新后的 Pod 开销的值：
```
Comment (xichengliudui) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:140]:
```suggestion
在 RuntimeClass 准入控制器进行修改后，你可以查看更新后的 PodSpec：
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 2]
```diff
@@ -310,24 +314,22 @@ This is 320 MiB, as expected:
...
-* [RuntimeClass](/docs/concepts/containers/runtime-class/)
-* [PodOverhead Design](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/688-pod-overhead)
+* Learn more about [RuntimeClass](/docs/concepts/containers/runtime-class/)
+* Read the [PodOverhead Design](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/688-pod-overhead)
+  enhancement proposal for extra context
 -->
-
-* [RuntimeClass](/zh/docs/concepts/containers/runtime-class/)
-* [PodOverhead 设计](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/688-pod-overhead)
+* 了解更多关于 [RuntimeClass](/zh/docs/concepts/containers/runtime-class/) 的信息
```
Comment (xichengliudui) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:334]: 了解 -> 学习

Comment (mengjiao-liu) [reply content/zh/docs/concepts/scheduling-eviction/pod-overhead.md]: Updated. Thanks for your review.

**Pull request merged**